internal/pkg/utils/testhelper/compilebin.go

```diff
@@ -0,0 +1,58 @@
+package testhelper
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/env"
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+)
+
+// CompileBinary compiles a binary used in the test by running a make command.
+func CompileBinary(t *testing.T, binaryName string, makeCommand string) string {
+	t.Helper()
```

Comment:
Pridal som tuto funkciu, povodne to bola metoda nad test runner, vid nizsie.

Test runner sluzi pre E2E testy, kde kazdy adresar je jeden test, to sa pouziva pre CLI, TA, BA.

Buffer worker je rieseny inak (kedze tam je to komplikovanejsie, ako len testovat API requesty), preto potrebujem tuto funkciu mimo test runner.